code/modules/customitems/item_defines.dm

```diff
@@ -426,6 +426,71 @@
 
 //////////// Masks ////////////
 
+obj/item/clothing/mask/gas/fluff/tyhailer // T0EPIC4U: Ty Omaha
+	name = "tacticool security gas mask"
+	desc = "A tacticool Security gas mask with modified 'Compli-o-nator 3000' device, let's Ty be the snowflake he is and choose the phrases it announces. It looks like it has some..residue..you probably don't want to touch it."
```

Comment:
a) Probably don't want to mention Ty specifically, as it's not really related to him backstory-wise. 
b) lets*
c) 😏